packages/core/src/loop/shared/inject-background-task-prompt.ts

```
@@ -0,0 +1,55 @@
+import type { LanguageModelV2Prompt } from '@ai-sdk/provider-v5';
+import {
+  generateBackgroundTaskSystemPrompt,
+  type AgentBackgroundConfig,
```

ESLint

[error] Prefer using a top-level type-only import instead of inline type specifiers.

(import/consistent-type-specifier-style)